python/GafferSceneTest/ShaderTest.py

```diff
@@ -361,5 +361,79 @@ def testNameSwitch( self ) :
 				[ network.Connection( network.Parameter( n, "", ), network.Parameter( "n3", "c" ) ) ]
 			)
 
+	def testSpline( self ) :
+
+		n1 = GafferSceneTest.TestShader( "n1" )
+
+		n2 = GafferSceneTest.TestShader( "n2" )
+		n2["type"].setValue( "test:surface" )
+
+		n2["parameters"]["i"].setInput( n1["out"]["r"] )
```

Comment:
What's the purpose of `n2` in this test? It seems largely redundant, since we only ever seem to assert anything about `n1` and `inN1`.

Reply:
Oh, yeah, I just copied that from another test before I fully understood how these tests were working.  Removed now.